erpnext/controllers/buying_controller.py

```diff
@@ -112,8 +112,8 @@ def update_valuation_rate(self, parentfield):
 					valuation_amount_adjustment -= item.item_tax_amount
 
 				self.round_floats_in(item)
-
-				item.conversion_factor = get_conversion_factor(item.item_code, item.uom).get("conversion_factor") or 1.0
+				if item.conversion_factor==0:
```

Comment:
Please update it and send the PR again, it will handle if conversion_factor is null.